app.rb

```diff
@@ -0,0 +1,196 @@
+require_relative "./lib/database_connection"
+require_relative "./lib/item_repository"
+require_relative "./lib/order_repository"
+require_relative "./lib/item"
+require_relative "./lib/order"
+require 'date'
+require 'colorize'
+
+class Application
+  def initialize(database_name, io, item_repository, order_repository)
+    DatabaseConnection.connect(database_name)
+    @io = io
+    @item_repository = item_repository
+    @order_repository = order_repository
+  end
+
+  def run
+    print_welcome
+    loop do
+      print_interface
+      user_selection(@io.gets.chomp)
```

Comment:
This is great, I added this but it made my RSpec crash so would love to discuss how you tested for this.